src/test/rustdoc-ui/private-doc-test.rs

```diff
@@ -0,0 +1,12 @@
+// check-pass
+
+#![deny(private_doc_tests)]
+
+mod foo {
+    /// private doc test
+    ///
+    /// ```ignore
+    /// assert!(false);
+    /// ```
```

Comment:
```
 tidy error: /checkout/src/test/rustdoc-ui/private-doc-test.rs:8: unexplained "```ignore" doctest; try one:

* make the test actually pass, by adding necessary imports and declarations, or
* use "```text", if the code is not Rust code, or
* use "```compile_fail,Ennnn", if the code is expected to fail at compile time, or
* use "```should_panic", if the code is expected to fail at run time, or
* use "```no_run", if the code should type-check but not necessary linkable/runnable, or
* explain it like "```ignore (cannot-test-this-because-xxxx)", if the annotation cannot be avoided.
```
```suggestion
    /// ```ignore (used for testing ignored doc tests)
    /// assert!(false);
    /// ```
```

Reply:
Tidy shouldn't check it here... Well, whatever, fixing it!